### PR TITLE
add finished and postponed objects for work orders

### PR DESCRIPTION
**Pull request by knorth55**
- line slots in alphabetical order
- add `finished-objects` slots
- subscribe work-order msg only once
  - current system only needs to subscribe work order once in the beginning.
- add `postponed-objects` in slots

### CODE REVIEW

File: jsk_arc2017_baxter/euslisp/lib/pick-interface.l [hunk 1]
```diff
@@ -248,6 +252,7 @@
           (ros::ros-info-green "[main] arm ~a: place object ~a in cardboard ~a" arm target-obj target-cardboard)
           (send self :update-json target-obj
                 :src (cons :bin target-bin) :dst (cons :cardboard target-cardboard))
+          (send self :add-finished-object arm target-obj)
```
Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/pick-interface.l]: Do we need `(send self :add-postponed-object arm target-obj)` in https://github.com/start-jsk/jsk_apc/pull/2296/files#diff-6a34c6b505e3c7331c663f31dbd8145bR247 ?

Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/lib/pick-interface.l]: here? no

Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/pick-interface.l]: Sorry for wrong place to comment.
I mean line 247 (`dropped` condition).

Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/lib/pick-interface.l]: Updated, thx!

**Comment by wkentaro** [comment]
can I merge?

**Comment by knorth55** [comment]
Yes

**Pull request merged**